data/json/terrain.json

```diff
@@ -8322,6 +8360,30 @@
     },
     "examine_action": "water_source"
   },
+  {
+    "type": "terrain",
+    "id": "t_water_dispenser",
+    "name": "water dispenser",
+    "description": "A machine with several taps that dispenses clean water.",
+    "symbol": "W",
+    "color": "light_blue",
+    "move_cost": 0,
+    "coverage": 40,
+    "flags": [ "TRANSPARENT" ],
+    "bash": {
+      "str_min": 15,
+      "str_max": 80,
+      "sound": "whack!",
+      "sound_fail": "thunk.",
+      "ter_set": "t_concrete",
+      "items": [
+        { "item": "steel_chunk", "count": [ 0, 2 ] },
+        { "item": "scrap", "count": [ 3, 6 ] },
+        { "item": "pipe", "count": [ 0, 2 ] }
+      ]
+    },
+    "examine_action": "clean_water_source"
```

Comment:
Unlimited clean water? Seems like OP.

Reply:
The actual main room of the robofac, where this spawns,  will remain inaccessible until mid-late game. By the time the player has access to the dispenser will mostly by a Qol thing. 

Thirst isnt an issue after the first day anyways.